### PR TITLE
Xcode 8 support

### PR DESCRIPTION
**Pull request by soffes**
Guessing you don't want to merge this yet, but thought I'd PR anyway so you have it if you want it down the road. Thanks for all of your work on this!

### CODE REVIEW

File: Source/SSLSecurity.swift [hunk 1]
```diff
@@ -139,9 +139,9 @@ public class SSLSecurity {
         }
         var policy: SecPolicyRef
         if self.validatedDN {
-            policy = SecPolicyCreateSSL(true, domain)
+            policy = SecPolicyCreateSSL(true, domain)!
```
Comment (soffes) [inline Source/SSLSecurity.swift]: I feel bad about this, but wanted to get it going quickly.

**Comment by daltoniam** [comment]
Thanks! Agreed, it should probably be in its own branch for now. I started working on a Swift 3 version here: 
https://github.com/daltoniam/Starscream/tree/swift3
That was before xCode 8 was out, just a nightly release build. I think there will need to be a few breaking changes to be inline with the Swift 3 changes (such as `writeString` and `writeData` just being `write` because of the first parameter now being include.) I'll make sure to cross reference your changes to see if I missed anything with the official swift 3/xCode 8. Thanks again.

**Comment by soffes** [comment]
@daltoniam no worries!

I made good progress here the other day: https://github.com/usecanvas/Starscream/tree/swift3

**Comment by daltoniam** [comment]
Ah very nice! I like those changes, would like to pull them when they are ready. We are getting much closer to a GM release so I am hoping to take some time soon to solidify everything for the Swift 3 release.

**Comment by soffes** [comment]
There's only 2 issues left. Both are related to pointer math. All of that is a little scary to me, so I'll let you take it from here if you want.

**Comment by daltoniam** [comment]
Fair enough. Thanks!

**Pull request closed without merging**